package.json

```diff
@@ -6,34 +6,37 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "npm run build-ts && node dist/app",
-    "serve": "node dist/app",
     "build-ts": "tsc",
-    "dev:ts": "nodemon src/app",
+    "dev": "nodemon src/app.ts",
     "lint": "eslint . --ext .ts"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/reuven12/Milestone3.git"
+    "url": "git+https://github.com/reuven12/Mileston4.git"
```

Comment:
It seems the configuration here was not done "naturally": was the package.json copied and then changed manually? 
The URL should have been more like `https://github.com/reuven12/MailStone4.git`. 
Make sure that in new projects you use `npm init` after cloning the empty github repo, and then the only parts in pachage.json allowed to paste inside are the scripts (also must be done carefully)